R/functions-Params.R

```diff
@@ -272,6 +274,52 @@ PeakGroupsParam <- function(minFraction = 0.9, extraPeaks = 1,
         subset = as.integer(subset), subsetAdjust = subsetAdjust)
 }
 
+#' @rdname adjustRtime
+LamaParama <- function(lamas = matrix(ncol = 2, nrow = 0,
+                                      dimnames = list(NULL, c("mz", "rt"))),
+                       method = c("loess", "gam"),
+                       span = 0.5,
+                       outlierTolerance = 3,
+                       zeroWeight = 10,
+                       ppm = 20,
+                       tolerance = 0,
+                       toleranceRt = 5,
+                       bs = "tp",
+                       rtMap = list(),
+                       nChromPeaks = numeric()) {
+    method <- match.arg(method)
+    if (method == "gam")
+        .check_gam_library()
+    if (is.data.frame(lamas))
+        lamas <- as.matrix(lamas)
+    if (ncol(lamas) != 2)
+        stop("the 'lamas' matrix needs to have two columns, composed of m/z, ",
+        "and retention time of the peaks from the reference dataset, in this ",
+        "order")
+    new("LamaParama", lamas = lamas,
+        method = method,
+        span = span,
+        outlierTolerance = outlierTolerance,
+        zeroWeight = zeroWeight,
+        ppm = ppm,
+        tolerance = tolerance,
+        toleranceRt = toleranceRt,
+        bs = bs,
+        rtMap = rtMap,
+        nChromPeaks = nChromPeaks)
+}
+
+#' Function to access rtMap from `LamaParama` object
```

Comment:
Am not completely happy with the name `rtMap` - it's OK if we name the slot like that, but to the user that might be completely unclear what we mean with that. A maybe more descriptive name might be better? ... didn't find anything though ... but then we should at least call it `rtimeMap`?

Reply:
You are right, what about sometimes with the word `match` or `matches` . 
Something like or `lamaMatches`, `lamaChrompeakMatches` or `matchedLamaChrompeaks`, `matchedRtime` to mimic the metaboAnnotation namings ?

Reply:
That sounds good to me !